src/main/tls.c

```diff
@@ -3851,6 +3852,23 @@ SSL_CTX *tls_init_ctx(fr_tls_server_conf_t *conf, int client, char const *chain_
 			SSL_CTX_set_default_passwd_cb(ctx, cbtls_password);
 		}
 	}
+	if (conf->private_key_password_file) {
+		FILE* passwordfile = fopen(conf->private_key_password_file, "r");
+		if (passwordfile) {
+			char password[256];
+			if(fgets(password, sizeof(password), passwordfile))	{
+				SSL_CTX_set_default_passwd_cb_userdata(ctx, password);
+				SSL_CTX_set_default_passwd_cb(ctx, cbtls_password);
+			}
+			else	{
```

Comment:
Random formatting is not recommended.  There's a consistent set of formatting in the server, for good reason.

Reply:
You are perfectly right, I tried to follow the existing style but it seems I was not doing a good job, sorry :-)
I just tried to improve it a bit.
I also tried to read https://wiki.freeradius.org/contributing/coding-standards#function-naming-conventions. 
If it is still not good, could you provide me where I could find the information about the formatting ?
Is there any formatting tool you might use ?

Reply:
Sorry, must have overlooked this one, I just improved on it.
Hope the formatting is now ok ?